akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/Behaviors.scala

```diff
@@ -155,6 +155,9 @@ object Behaviors {
    * This type of Behavior wraps another Behavior while allowing you to perform
    * some action upon each received message or signal. It is most commonly used
    * for logging or tracing what a certain Actor does.
+   *
+   * If a wrapped behavior returns a tap itself, the new tap replaces the existing one,
+   * to protect against stack overflow due to recursively adding taps.
```

Comment:
Ups. Not true, I created a new unique identifier for each tap invocation.

Reply:
right, for `withTimers` we know that there should only be one

Reply:
Yep, with `tap` I wasn't sure what is the right way, so left a FIXME that it could lead to SO